docs/source/developer/listings.rst

```diff
@@ -0,0 +1,7 @@
+Listings
+========
+
```

Comment:
This section definitely needs a more descriptive title, and a paragraph explanation of what the reader is looking at.